### PR TITLE
FEATURE: adds support for tagsList

### PR DESCRIPTION
**Pull request by jjaffeux**
Usage:

```
[wrap=template key=foo tagsList=bar,baz]
Something
[/wrap]
```

Note this commit also improves how dates are displayed (using local dates).

### CODE REVIEW

**Comment by jjaffeux** [comment]
Capture d’écran 2020-10-22 à 10 54 26
Capture d’écran 2020-10-22 à 10 54 32

File: common/common.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .d-wrap[data-wrap="template"] {
   padding: 0.5em;
-  border: 1px dashed $primary-medium;
+  border: 5px solid var(--primary-low, $primary-low);
```
Comment (CvX) [inline common/common.scss:3]: Hm, it's `var(--primary-low, $primary-low)` but just `var(--primary-low)` below? Should probably stick to one version?

File: javascripts/discourse/initializers/init-discourse-reply-template-component.js.es6 [hunk 2]
```diff
@@ -23,48 +23,105 @@ function buildButton(dataset, extraClass) {
...
+    timezone: moment.tz.guess()
+  };
+
+  if (!date.startOf("day").isSame(date) && !date.endOf("day").isSame(date)) {
+    options.time = date.format("HH:mm");
+  }
+
+  const parts = [];
+  Object.keys(options).forEach(key => {
+    const value = options[key];
```
Comment (CvX) [inline javascripts/discourse/initializers/init-discourse-reply-template-component.js.es6:63]:
```suggestion
  Object.entries(options).forEach(([key, value]) => {
```

Or you could even go with `parts = Object.entries().map();`

File: common/common.scss (hunk 1 above)
Comment (jjaffeux) [reply common/common.scss:3]: yes I should add fallback here too :+1: thanks

File: javascripts/discourse/initializers/init-discourse-reply-template-component.js.es6 [hunk 3]
```diff
@@ -73,8 +130,20 @@ function openComposerWithTemplateAndAction(controller, post, dataset) {
         match[1] = match[1].replace(replacer.regex, replacer.fn);
       });
 
+      const tags = [];
+      wrap
+        .querySelectorAll("input[type=checkbox]:checked")
+        .forEach(checkbox => {
+          tags.push(`#${checkbox.value}`);
+        });
```
Comment (CvX) [inline javascripts/discourse/initializers/init-discourse-reply-template-component.js.es6:138]: Similarly to the other one:

```suggestion
      const tags = wrap
        .querySelectorAll("input[type=checkbox]:checked")
        .values()
        .map((checkbox) => `#${checkbox.value}`);
```

**Review by CvX — APPROVED** [review]

**Pull request merged**